project.md

```diff
@@ -11,41 +11,64 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+A DSL for avid sporting parents and the sporting community that allows for quick and better stats keeping for sporting events. The typical sporting parent who wants to keep track of all their kids stats or even high school coaches that are responsable for keeping track of stats for games currently use a peice of paper, a notes app, excell and varios webapages to help keep track of stats for sporting events. Now imagine a DSL that allowed for parents and coaches to easilly add one stat per player at a time in live action and after the game is over have a collection a each players stats and an accumlitive stat sheet representative of the whole team. Moreover, imagine this DSL could be used for highschool through college level programs that currently manually enter stats into their programs webpage. Now they can use the DSL in real time and automaitically tranfer the data to their programs webpage without any manual labor after the game has ended. Other uses of such DSL could also be for updating and broadcasting live stats per game (providing a play by play) for people who cannot attend the game in person. For this Dsl I will focus on Football since it is the sport I know best.
+
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A dsl that could be accessable during game time would allow for users to update statistics in real time. The need for real time statisic updates at the non profesional level is fairly high. It would be amazing if such a laguage could be used to provide accurate statistics for players coaches and family members in real time. As well as get rid of the need to record the statisic and maunally input each stat into some interface later on. 
+
```

Comment:
A concern I have is that users won't be expected to know how to program based on the background you've provided. This is fine, I think it allows you to design something more useful for a broader audience. However, it will be crucial to keep in mind that users might not know about standard programming principles like defining variables and calling functions.